README.md

```diff
@@ -112,9 +112,11 @@ automatically at build time.
 ## Testing
 
 Test assumes there is a running AAOS VM. Again, make sure you also have ADB installed.
+Also, tests may interfere when run in parallel. To avoid false negatives, make
+sure you force a single thread.
 
 ```shell
-$ ADB_PATH=/path/to/adb/bin cargo test
+$ ADB_PATH=/path/to/adb/bin cargo test --lib -- --test-threads=1
```

Comment:
Why need this extra `--` in the middle? Immediately after the `lib`'...
Maybe there is some style or convention I'm not aware about :)

Reply:
It is to bypass test arguments and send them to cargo. It is documented [here](https://web.mit.edu/rust-lang_v1.25/arch/amd64_ubuntu1404/share/doc/rust/html/book/second-edition/ch11-02-running-tests.html#running-tests-in-parallel-or-consecutively) .